src/components/router/Router.brs

```diff
@@ -51,22 +48,45 @@ sub resetHistory(rootPath = "" as String)
   m._history = []
 
   if (rootPath <> "")
-    m._history.push({ path: rootPath, params: {} })
+    rootRoute = CreateObject("roSGNode", "ActivatedRoute")
+    rootRoute.setFields({ path: rootPath })
+
+    m._history.push(rootRoute.getFields())
```

Comment:
`_createHistoryItem` seems to be missing

```suggestion
    m._history.push(_createHistoryItem(rootRoute.getFields()))
```

Reply:
by missing I meant to keep the consistent interface of the objects stored in the history 😊